### ISSUE TITLE
scylla_post_install.sh: fail to get a valid (integer) memory size

### ISSUE THREAD

**Issue by amoskong** [scylladb/scylla#5569]
*Installation details*
Scylla version (or git commit hash): scylla_666.development-0.20200107.6e0a073b2e5-1_amd64
Cluster size: 1
OS (RHEL/CentOS/Ubuntu/AWS AMI): Debian 10

### ERROR in installation: /opt/scylladb/scripts/scylla_post_install.sh: line 62: [: 3.87622e+09: integer expression expected

```
scylla-test@amos-debian10:~$ sudo apt-get install scylla
....
....
Setting up scylla-tools-core (666.development-20200108.76275abbacd-1) ...
Setting up scylla-tools (666.development-20200108.76275abbacd-1) ...
Setting up scylla-server (666.development-0.20200107.6e0a073b2e5-1) ...
scylla:x:108:114::/var/lib/scylla:/usr/sbin/nologin
scylla:x:114:
/opt/scylladb/scripts/scylla_post_install.sh: line 62: [: 3.87622e+09: integer expression expected
Setting up scylla-jmx (666.development-20200108.4c8660d41ad-1) ...
Setting up scylla (666.development-0.20200107.6e0a073b2e5-1) ...

```

###

```
scylla-test@amos-debian10:~$ cat /opt/scylladb/scripts/scylla_post_install.sh -n

        58	    # For systems with not a lot of memory, override default reservations for the slices
        59	    # seastar has a minimum reservation of 1.5GB that kicks in, and 21GB * 0.07 = 1.5GB.
        60	    # So for anything smaller than that we will not use percentages in the helper slice
>>>>>   61	    MEMTOTAL_BYTES=$(cat /proc/meminfo | grep MemTotal | awk '{print $2 * 1024}')
        62	    if [ $MEMTOTAL_BYTES -lt 23008753371 ]; then
        63	        mkdir -p /etc/systemd/system/scylla-helper.slice.d/
        64	        cat << EOS > /etc/systemd/system/scylla-helper.slice.d/memory.conf
```

A suggest fix:

```
scylla-test@amos-debian10:~$ cat /proc/meminfo | grep MemTotal | awk '{str=sprintf("%d", $2 * 1024); print str}'
2147483647
```

@syuu1228

**Comment by syuu1228** [comment]
@slivne The fix 5627888b7c82745854ae06b9c3b8e69450ce236e does not merged into branch-3.2, so I'm still getting same error on 3.2.1 package:
```
Setting up scylla-server (3.2.1-0.20200122.e3e301906d5-1) ...
Installing new version of config file /etc/init/scylla-fstrim.conf ...
Installing new version of config file /etc/init/scylla-housekeeping-daily.conf ...
Installing new version of config file /etc/init/scylla-housekeeping-restart.conf ...
Installing new version of config file /etc/init/scylla-server.conf ...
scylla:x:106:107::/var/lib/scylla:/usr/sbin/nologin
scylla:x:107:
/opt/scylladb/scripts/scylla_post_install.sh: line 62: [: 6.73532e+10: integer expression expected

```

**Comment by avikivity** [comment]
Already backported; removing label.
